Basic-Scripts/balanced_paranthesis.py

```diff
@@ -0,0 +1,40 @@
+openBracketList = ["[", "{", "("]
+closeBracketList = ["]", "}", ")"]
+
+
+def checkParentheses(data: str) -> str:

```

Comment:
Function names should be lowercase, with words separated by underscores as necessary to improve readability.

Reply:
As already mentioned i am using camel case naming convention for variables and functions. Please specify which naming convention you want me to follow?

Reply:
by the way i have made the changes.

Reply:
Follow PEP8 standard for naming convention
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names